prime-router/src/main/kotlin/history/azure/DatabaseSubmissionsAccess.kt

```diff
@@ -124,17 +128,33 @@ class DatabaseSubmissionsAccess(
         )
     }
 
-    override fun <T> fetchRelatedActions(actionId: Long, klass: Class<T>): List<T> {
-        val cte = reportDescendantExpression(actionId)
+    override fun <T> fetchRelatedActions(reportId: UUID, klass: Class<T>): List<T> {
```

Comment:
Please add doc to explain what this is doing.

Reply:
The kdoc gets inherited from the base class